README.md

```diff
@@ -2,198 +2,66 @@
-  - `pip install -r requirements.txt`
-4. Setup the database:
-  - Install [PostgreSQL](https://www.postgresql.org/download/) if needed
-  - [Create a database](https://www.postgresql.org/docs/10/static/tutorial-createdb.html).
-5. Create a main postgresql use with password:
-  ```shell
-  $> sudo su postgres
-  $> psql
-  $ CREATE USER backend WITH PASSWORD 'backend' CREATEDB CREATEROLE SUPERUSER;
-  ```
-6. Create two databases for both orgs: owkin and chu-nantes. A shell script is available, do not hesitate to run it.
-It will drop the databases if they are already created, then create them and grant all privileges to your main user backend.
- (If this is the first time you create the databases, you will see some warnings which are pointless):
-
-  ```shell
-  $> ./scripts/recreate_db.sh
-```
-7. We will populate data:
+## Running a development instance
 
-###### Clean environment (recommanded)
+For the local installation of substra-backend (and companions), please refer to the [setup documentation](https://doc.substra.ai/setup/local_install_skaffold.html).
 
-- With django migrations
-```shell
-BACKEND_ORG=owkin BACKEND_DEFAULT_PORT=8000 python backend/manage.py migrate --settings=backend.settings.dev
-BACKEND_ORG=chu-nantes BACKEND_DEFAULT_PORT=8001 python backend/manage.py migrate --settings=backend.settings.dev
+With [hlf-k8s]() already running and requirements fulfilled, this should boils down to:
+```sh
+skaffold dev
 ```
 
-###### With fixtures (fixtures container has been run from hlf-k8s, old behavior for testing)
-
-data in fixtures are relative to the data already set in the ledger if the fixtures container instance succeeded
+## Django management
 
-Two solutions:
-- With django migrations + load data
-```shell
-BACKEND_ORG=owkin BACKEND_DEFAULT_PORT=8000 python backend/manage.py migrate --settings=backend.settings.dev
-BACKEND_ORG=chu-nantes BACKEND_DEFAULT_PORT=8001 python backend/manage.py migrate --settings=backend.settings.dev
-BACKEND_ORG=owkin BACKEND_DEFAULT_PORT=8000 python backend/manage.py loaddata ./fixtures/data_owkin.json --settings=backend.settings.dev
-BACKEND_ORG=chu-nantes BACKEND_DEFAULT_PORT=8001 python backend/manage.py loaddata ./fixtures/data_chu-nantes.json --settings=backend.settings.dev
-```
-- From dumps:
-```shell
-  $> ./scripts/populate_db.sh
+To get access to the Django management tool, spawn a shell in the running container:
+```sh
+kubectl -n org-1 exec -i -t $(kubectl -n org-1 get pods -l=app.kubernetes.io/name=substra-backend-server -o name) -c substra-backend -- /bin/bash
 ```
-If you don't want to replicate the data in the ledger, simply run the django migrations.
 
-Populate media files
-```shell
-  $> ./scripts/load_fixtures.sh
-```
-It will clean the `medias` folders and create the `owkin` and `chu-nantes` folders in the `medias` folder.
+This will also gives you access to the celery CLI.
```

Comment:
Could you provide a simple example of useful celery command here (like celery list)?